hudi-common/src/main/java/org/apache/hudi/common/model/HoodieCommitMetadata.java

```diff
@@ -109,6 +108,17 @@ public void setCompacted(Boolean compacted) {
     return filePaths;
   }
 
+  public HashMap<String, String> getFileIdWithoutSuffixAndRelativePaths() {
```

Comment:
can this sit somewhere else? in a helper method?

Reply:
done